libs/block-template-custom-element/README.md

```diff
@@ -8,11 +8,38 @@ See [https://blockprotocol.org/docs/developing-blocks](https://blockprotocol.org
 
 Other templates are available. See `npx create-block-app@latest --help`
 
-## Step two: write and build a component
+## Step two: decide on and build the entity type for your block
+
+A key part of the Block Protocol is the use of types to describe the data your block will work with.
+
+Your block should be associated with an “entity type” which will be used by embedding applications
+to understand what sorts of entities can be sent to it (e.g. what properties do they have?).
+
+You can create an entity type on [blockprotocol.org](https://blockprotocol.org) — see [the docs](https://blockprotocol.org/docs/developing-blocks) for a full guide.
+
+Once you have created the type representing the data your block needs, copy its URL for use in the next step.
+
+## Step three: update your block's metadata
 
 1.  Change into the folder: `cd path/to/your-block-name`
 
-1.  Update the metadata in package.json to change the default `tagName` under `blockprotocol.blockType`
+1.  Update the `blockprotocol` metadata object in package.json:
+
+- set `schema` to the URL of the entity type you created in the previous step
+- change the default `tagName` under `blockType` to the tag for your element
+- optionally update additional fields which will be used to identify and describe the block when published:
+  - `displayName`: a friendly display name
+  - `examples`: an array of example data structures your block would accept and use
+  - `image`: a preview image showing your block in action (in place of `public/block-preview.png`)
+  - `icon`: an icon to be associated with your block (in place of `public/omega.svg`)
+  - `name`: a slugified name for your block (which may differ to the package `name` in package.json)
+    - this may either be in the format `slug` or `@namespace/slug` where `namespace` is your blockprotocol.org username
+
+1.  Run `yarn schema` to automatically TypeScript types from your block's entity type / `schema`
```

Comment:
to automatically generate, right?

Reply:
Yep, thanks! https://github.com/blockprotocol/blockprotocol/pull/986/commits/f9f0a257aea9593775afa9d72b41ba01ad561530